src/DocBlock/Tags/InvalidTag.php

```diff
@@ -80,29 +81,12 @@ private function flattenExceptionBacktrace(Throwable $exception) : void
         $traceProperty = (new ReflectionClass(Exception::class))->getProperty('trace');
         $traceProperty->setAccessible(true);
 
-        $flatten =
-            /** @param mixed $value */
-            static function (&$value) : void {
-                if ($value instanceof Closure) {
-                    $closureReflection = new ReflectionFunction($value);
-                    $value             = sprintf(
-                        '(Closure at %s:%s)',
-                        $closureReflection->getFileName(),
-                        $closureReflection->getStartLine()
-                    );
-                } elseif (is_object($value)) {
-                    $value = sprintf('object(%s)', get_class($value));
-                } elseif (is_resource($value)) {
-                    $value = sprintf('resource(%s)', get_resource_type($value));
-                }
-            };
-
         do {
             $trace = $exception->getTrace();
             if (isset($trace[0]['args'])) {
                 $trace = array_map(
-                    static function (array $call) use ($flatten) : array {
-                        array_walk_recursive($call['args'], $flatten);
+                    function (array $call) : array {
+                        $call['args'] = array_map([$this, 'flattenArguments'], $call['args']);
```

Comment:
Give the php version requirements I would recommend (not tested):
https://github.com/phpDocumentor/ReflectionDocBlock/blob/master/composer.json#L17
```suggestion
                        $call['args'] = array_map(function($value) { return $this->flattenArguments($value); }, $call['args']);
```

Reply:
👎 for changing this. The array-callable syntax is much shorter and clearer?

Reply:
Nope, for me this syntax is "compilation" compatible and explicit for any code searcher or analyser :)
Don't you agree?

Reply:
And that would remove the need of `ref="SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod">`

Reply:
Both can be formally detected. I guess it just hasn't been implemented by that code style package. psalm itself certainly recognises uses like that, in it's core stuff.